easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2016.04.eb

```diff
@@ -0,0 +1,22 @@
+name = 'OpenSSL'
+version = '1.0.1p'
+
+homepage = 'http://www.openssl.org/'
+description = """The OpenSSL Project is a collaborative effort to develop a robust,
+ commercial-grade, full-featured, and Open Source toolchain implementing the Secure
+ Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) protocols as well
+ as a full-strength general purpose cryptography library."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```

Comment:
@AFCJamie please drop the `opt` and `optarch` entries here, they're enabled by default